eemeter/cli.py

```diff
@@ -149,35 +157,105 @@ def run_meter(project, trace_object):
         project['project_start'],
         project['project_end']
     )
+
     ee = EnergyEfficiencyMeter()
+
+    if options is not None and \
+       'ignore_data_sufficiency' in options.keys() and \
```

Comment:
Usually I do 8 spaces instead of 3 for a continued if, so just to be consistent let's change this to match. Pep 8 is a little unclear on this but I usually follow this guidance:

```
# Add some extra indentation on the conditional continuation line.
if (this_is_one_thing
        and that_is_another_thing):
    do_something()
```
This seems as good a place as any to note the other minor flake8 errors that are causing issues in the travis build that need to be fixed before everything will go green.

```
eemeter/cli.py:16:1: F401 'eemeter.modeling.models.caltrack_daily.CaltrackDailyModel' imported but unused
eemeter/cli.py:165:43: E712 comparison to True should be 'if cond is True:' or 'if cond:'
eemeter/cli.py:168:14: E111 indentation is not a multiple of four
eemeter/cli.py:171:1: W293 blank line contains whitespace
eemeter/cli.py:173:1: W293 blank line contains whitespace
eemeter/cli.py:293:16: E201 whitespace after '{'
eemeter/cli.py:293:67: E202 whitespace before '}'
eemeter/cli.py:293:69: W291 trailing whitespace
```